getting-started/vs-code-integration/dpl-wizard.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Configure Document Processing Libraries 
+page_title: Document Processing Libraries Integration
+description: Learn how to add a reference to the Telerik Document Processing Libraries(DPL) packages in an existing Telerik Ui for Blazor application using the Document Processing Libraries Wizard.
+slug: getting-started-vs-code-integration-dpl
+position: 5
+---
+
+# Configuring Document Processing Libraries
+
+This article demonstrates how to add a reference to the [Telerik Document Processing Libraries (DPL)](https://www.telerik.com/document-processing-libraries) packages in an existing Telerik UI for Blazor application using the Document Processing Libraries Wizard. The wizard is included in the [Telerik UI for Blazor Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=TelerikInc.blazortemplatewizard) and enables you to easily add the required packages when working with the Telerik Document Processing Library.
+
+## Getting the Wizard
+
+To use the Document Processing Libraries Wizard, install the [Telerik UI for Blazor Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=TelerikInc.blazortemplatewizard).
+
+## Starting the Wizard
+
+`Using the File Explorer Context Menu`
+
+To use **Document Processing Libraries Configuration Wizard** from the **VS Code**'s file explorer's context menu:
+ - Right-click on a file in the files tree.
+ - Select the `Telerik UI for Blazor: Add Document Processing Libraries`.
+
+  ![Configure Document Processing Wizard, Context Menu](images/DPLBlazor_ContextMenu.png)
+
+`Using the Code Editor Command Palette`
+
+To use **Document Processing Libraries Configuration Wizard** from the **VS Code** command palette:
+
+ - Open a `.razor` file or a `Razor` language in the code editor.
+ - Select the `Telerik UI for Blazor: Add Document Processing Libraries`.
+
+ ![Configure Document Processing Wizard, Command Palette](images/DPLBlazor_Pallete.png)
+
+After executing one of the options the wizard must appear like this: 
+
+ ![Configure Document Processing Wizard](images/DPLWizardBlazor.png "Configure Document Processing Wizard")
+
+ The Telerik Document Processing Configuration wizard allows you to select which libraries your project would use. Also have options to use `Dev` or `Trial` license and selection of the `.csproj` file, in which you want the installation to be provded. This option is useful in cases, when the user have many opened projects in the workspace. If a library you selected has any dependencies, they are automatically checked. In the table below there is a list of the assemblies that will be selected for each library:
```

Comment:
I like Dimo's suggestion a lot. However, consider placing the `##` heading as per my earlier recommendation instead.

Reply:
`Telerik Document Processing Configuration wizard` is not really the correct name. It should either be `Telerik Document Processing Libraries wizard` (as per what the tab name says) or `Configure Document Processing Libraries wizard` (as per the large title within the wizard itself. I recommend going with the tab name instead (since it contains Telerik in it).